### PR TITLE
Feature/cplp 1498/assign user role

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**

### CODE REVIEW

File: cx-portal-shared-components/src/components/basic/Table/index.tsx [hunk 1]
```diff
@@ -86,6 +86,11 @@ export const Table = ({
     onSelection,
     searchExpr,
   }
+
+  const handleOnCellClick = (params: any) => {
+    onSelection && onSelection([params.row.companyUserId])
```
Comment (lavanya-bmw) [inline cx-portal-shared-components/src/components/basic/Table/index.tsx:91]: On all tables cellClick, I don't think params.row.companyUserId will be used in shared component as it is not dynamic, please check this

File: cx-portal/src/components/pages/AppUserManagement/index.tsx [hunk 2]
```diff
@@ -52,6 +75,22 @@ export default function AppUserManagement() {
         />
       )}
       <AppUserDetailsTable />
+      <PageSnackbar
+        contactLinks=""
+        contactText=""
+        description={
+          userRoleResponse === 'success'
+            ? 'User Role assigned successfully'
+            : 'Error while assigning user role'
```
Comment (lavanya-bmw) [inline cx-portal/src/components/pages/AppUserManagement/index.tsx:84]: Please get the description text from translation

**Review by lavanya-bmw — CHANGES_REQUESTED** [review]
Please add release notes for this PR.

File: cx-portal-shared-components/src/components/basic/Table/index.tsx (hunk 1 above)
Comment (nidhigarg-bmw) [reply cx-portal-shared-components/src/components/basic/Table/index.tsx:91]: This is the data which we will get dynamic and according to particular row. This function fetch the particular row of table.

**Review by lavanya-bmw — APPROVED** [review]
lgtm

**Pull request merged**